app/controllers/email_actions_controller.rb

```diff
@@ -19,19 +19,13 @@ def mark_discussion_as_read
   end
 
   def mark_summary_email_as_read
-    @inbox = Inbox.new(user)
-    @inbox.load
-    time_start = Time.at(params[:time_start].to_i).utc
+    time_start  = Time.at(params[:time_start].to_i).utc
     time_finish = Time.at(params[:time_finish].to_i).utc
 
-    Queries::VisibleDiscussions.new(user: user,
-                                    groups: user.inbox_groups).
+    Queries::VisibleDiscussions.new(user: user).
```

Comment:
Hm, so this came out of a desire to get rid of the concept of `inbox_groups`, which used the concept of `memberships.inbox_position`, which isn't used.

I'll take a look in the DB about the nature of the current state of inbox_position; maybe I can just back this out and leave the `inbox_groups` method on user for now.